appveyor.yml

```diff
@@ -0,0 +1,49 @@
+build: false
+platform:
+  - x64
+clone_folder: c:\projects\workspace
+
+environment:
+  matrix:
+  - dependencies: lowest
+    php_ver_target: 7.0
+  - dependencies: highest
+    php_ver_target: 7.0
+  - dependencies: lowest
+    php_ver_target: 7.1
+  - dependencies: highest
+    php_ver_target: 7.1
+
+cache:
+    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
```

Comment:
It's not strictly necessary to cache anything other than composer, but Humbug is the only project likely to benefit from Windows testing in the org, so we have cache to burn on PHP installs.